onboarding/src/Components/hooks/QnAAPI.js

```diff
@@ -0,0 +1,92 @@
+import { getPath, getCookie } from "../utils.js";
+
+const url = getPath();
+const token = getCookie("csrftoken");
```

Comment:
Is it elegant to assign and use global variables in a component like here?
Seriously 'speaking'
maybe we should think about to assign those constantly repeated variables somewhere for whole session.

Reply:
Why it would be not elegant? If they are repeating, it's easier to handle them in one place.
I would only change names of this constants to upper case. And maybe I would do extra function like this:
```js
function makeRequest(url, method, body){
  const jsonBody = body ? JSON.stringify(body) : undefined;
  const response = fetch(url, {
    method: method,
    headers: {
      "Content-Type": "application/json",
...
    },
    body: jsonBody
  });
  if(!response.ok) {
    throw new Error("Something went wrong");
  }
}```